backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -11,14 +11,13 @@
 import javax.persistence.EnumType;
 import javax.persistence.Enumerated;
 import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
 import javax.persistence.OneToMany;
 import java.time.LocalDate;
 import java.util.List;
 
 
 @Entity
-public class Person extends EternalEntity {
+public class Person extends OrganizationScopedEternalEntity {
```

Comment:
Do we want to expose the Organization as part of the GraphQL schema as well?

Reply:
Not in general, no—in graphql-land, you should only ever be querying within a specific organization. The reason we have organization on every object, even if it's implied by other objects (e.g. TestOrder having a person _and_ an organization) is so that we can implement row-level security on the database side.

Reply:
I was thinking about this and forgot to write down my thought: it is probably worth having some form of tagging interface or at least javadoc that explains the above point inline. I will add that.